_posts/2024-07-23-WildFly33-Released.adoc

```diff
@@ -0,0 +1,248 @@
+
+link:https://github.com/wildfly/certifications/blob/MP6.1/WildFly_33.0.0.Final/microprofile-6.1-selected-specifications/microprofile-6.1-selected-specifications-certification.adoc[Compatibility evidence] for the above specifications that are part of MicroProfile 6.1 can be found in the WildFly Certifications repository on GitHub.
+
+[[preview-ee11-se17]]
+=== WildFly Preview, EE 11 and SE 17
+
+As I noted in the https://www.wildfly.org/news/2024/04/25/WildFly32-Released/[WildFly 32 release announcement], beginning with that release we are using WildFly Preview to provide a look at what we're doing for Jakarta EE 11 support.  EE 11 won't go GA before later this year, and we don't expect standard WildFly to support EE 11 before the WildFly 35 release. But there are milestone, Release Candidate and Final releases of many EE 11 specs and implementations available, so we are providiong those in WildFly Preview. This means for a number of EE APIs, WildFly Preview no longer provides an EE 10 compatible implementation.
+
+However, for a number of specifications that are planning changes for EE 11 we are still offering the EE 10 variant. In future releases we'll shift those to the EE 11 variants.
+
+As a result of this shift to EE 11 APIs, *WildFly Preview no longer supports running on Java SE 11.* Going forward, if you want to use WildFly Preview you'll need to use SE 17 or higher.  A number of EE 11 APIs no longer produce SE 11 compatible binaries, which means an EE 11 runtime can no longer support SE 11.
+
+The following table lists the various Jakarta EE technologies offered by WildFly Preview 33, along with information about which EE platform version the specification relates to. Note that a number of Jakarta specifications are unchanged between EE 10 and EE 11, while other EE technologies that WildFly offers are not part of EE 11.
+
+[cols=",,",options="header"]
+|=======================================================================
+|Jakarta EE Technology |WildFly Preview Version| EE Version
+
+|Jakarta Activation| 2.1 |10 & 11
+
+|Jakarta Annotations| 3.0.0 |11
+
+|Jakarta Authentication| 3.0 |10
+
+|Jakarta Authorization| 3.0.0 |11
+
+|Jakarta Batch| 2.1 |10 & 11
+
+|Jakarta Concurrency| 3.1.0 |11
+
+|Jakarta Connectors| 2.1 |10 & 11
+
+|Jakarta Contexts and Dependency Injection| 4.1.0 |11
+
+|Jakarta Debugging Support for Other Languages| 2.0 |10 & 11
+
+|Jakarta Dependency Injection| 2.0 |10 & 11
+
+|Jakarta Enterprise Beans| 4.0 |10 & 11
+
+|Jakarta Enterprise Web Services| 2.0 |10 xref:note1[^1^]
+
+|Jakarta Expression Language| 6.0.0 |11
+
+|Jakarta Faces| 4.1.0 |11
+
+|Jakarta Interceptors| 2.2.0 |11
+
+|Jakarta JSON Binding| 3.0 |10 & 11
+
+|Jakarta JSON Processing| 2.1 |10 & 11
+
+|Jakarta Mail| 2.1 |10 & 11
+
+|Jakarta Messaging| 3.1 |10 & 11
+
+| Jakarta MVC
+(_preview stability only_)| 2.1| N/A xref:note2[^2^]
+
+|Jakarta Pages| 3.1 |10
+
+|Jakarta Persistence| 3.2.0 |11
+
+|Jakarta RESTful Web Services| 3.1 |10
```

Comment:
This should be 4.0 and 11 for the EE version.

Reply:
Thanks; fixed.